taxi/views.py

```diff
@@ -1,3 +1,16 @@
+from django.http import HttpResponse
 from django.shortcuts import render
 
-# Create your views here.
+from taxi.models import Driver, Manufacturer, Car
+
+
+def index(request):
+    num_drivers = Driver.objects.count()
+    num_manufacts = Manufacturer.objects.count()
+    num_cars = Car.objects.count()
+    context = {
+        "num_drivers": num_drivers,
+        "num_manufacts": num_manufacts,
+        "num_cars": num_cars,
+    }
```

Comment:
do not repeat yourself?

Reply:
i don't think is has something to do with DRY principle, it's just a regular code. As i saw in videos, Dania did it the exact same way. Maybe mentors will say something about this.

Reply:
np